openpype/tools/pyblish_pype/control.py

```diff
@@ -22,7 +22,7 @@
 from . import util
 from .constants import InstanceStates
 
-from openpype.api import get_project_settings
+from openpype.settings import get_project_settings
```

Comment:
Could be replaced with `from openpype.settings import get_current_project_settings` looking at the code down below. No?

Reply:
`get_current_project_settings` is not the best function available in settings api. But it is probably better to use it if env is already used in code.